### PR TITLE
Added bem-cn project

### PR DESCRIPTION
**Pull request by albburtsev**

### CODE REVIEW

File: projects/projects.en.md [hunk 1]
```diff
@@ -109,5 +109,13 @@ Project repository on [GitHub](https://github.com/bem/bem-site-engine).
 
 Authored by Andrey Kuznetsov [gh: tormozz48](https://github.com/tormozz48), [@kuznetsov48](https://twitter.com/@kuznetsov48), [andrey.kuznetsov48@yandex.ua](mailto:andrey.kuznetsov48@yandex.ua).
 
+## bem-cn
+
+Friendly BEM class names generator. Simple syntax originally was created for using in React-components, but it is good enough with any other frameworks.
```
Comment (mursya) [inline projects/projects.en.md]: was created originally to be used in React components

File: projects/projects.ru.md [hunk 2]
```diff
@@ -104,5 +104,13 @@
 
 Автор проекта –  Андрей Кузнецов, [gh: tormozz48](https://github.com/tormozz48), [@kuznetsov48](https://twitter.com/@kuznetsov48), [andrey.kuznetsov48@yandex.ua](mailto:andrey.kuznetsov48@yandex.ua).
 
+## bem-cn
+
+Генератор имен классов. Лаконичный синтаксис изначально предназначался для использования в React-компонентах. Однако, утилита может использоваться с другими фреймворками.
```
Comment (mursya) [inline projects/projects.ru.md]: Генератор имен классов по методологии БЭМ. 
и с другими фреймворками

File: projects/projects.en.md [hunk 3]
```diff
@@ -109,5 +109,13 @@ Project repository on [GitHub](https://github.com/bem/bem-site-engine).
...
+
+Project repository on [GitHub](https://github.com/albburtsev/bem-cn).
```
Comment (tadatuta) [inline projects/projects.en.md]: Project repository **is** on

**Comment by tadatuta** [comment]
@albburtsev thank you!

**Pull request merged**